Sources/ValidatingProperty.swift

```diff
@@ -178,26 +178,6 @@ public final class ValidatingProperty<Value, ValidationError: Swift.Error>: Muta
 		self.init(MutableProperty(initial), with: other, validator)
 	}
 
-	/// Create a `ValidatingProperty` that presents a mutable validating
-	/// view for an inner mutable property.
-	///
-	/// The proposed value is only committed when `valid` is returned by the
-	/// `validator` closure.
-	///
-	/// - note: `inner` is retained by the created property.
-	///
-	/// - parameters:
-	///   - inner: The inner property which validated values are committed to.
-	///   - other: The property that `validator` depends on.
-	///   - validator: The closure to invoke for any proposed value to `self`.
-	public convenience init<U, E>(
```

Comment:
The capture of the compiler error

<img width="1036" alt="2018-06-06 14 35 28" src="https://user-images.githubusercontent.com/909674/41066617-46a1fc9e-6997-11e8-804b-3fcc984be5a3.png">

Reply:
The following overload will be used instead after this removal: https://github.com/ReactiveCocoa/ReactiveSwift/blob/7c25bc2e284ba54a8d81d3843a59cdd933c8f823/Sources/ValidatingProperty.swift#L128-L132

Reply:
I think this makes sense 👍